<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>Robot</string>
    </property>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>401</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>Robot Position</string>
     </property>
     <widget class="QLineEdit" name="robotPosX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="robotPosY">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="robotPosZ">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="robotAngleA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="robotAngleB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="robotAngleC">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="robotAngleD">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>191</width>
        <height>55</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Robot Speed</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="robotSpeedText"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>401</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>Robot Ctrl</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="robotStartBtn">
         <property name="text">
          <string>Robot Start</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="robotStopBtn">
         <property name="text">
          <string>Robot Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_17">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="robotStatusIndicationLedLay"/>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Robot Status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="arsCalibStartBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>129</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Calibrate Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="arsHomeStartBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>129</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_19">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="robotCalibrationStatusIndicationLedLay"/>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Calib Status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>351</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>File Mode</string>
    </property>
    <widget class="QPushButton" name="arsFileModeStartBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>File Mode Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="arsFileModeStopBtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>File Mode Stop</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_21">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="robotStatusIndicationLedLay_4"/>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>File Mode Status</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Jog Mode</string>
    </property>
    <widget class="QPushButton" name="arsJogModeStartBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Jog Mode Start</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_18">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="robotStatusIndicationLedLay_2"/>
    </widget>
    <widget class="QPushButton" name="arsJogModeStopBtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Jog Mode Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Robot Jog Mode Status</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Teach Mode</string>
    </property>
    <widget class="QPushButton" name="arsTeachModeStartBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Teach Mode Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="arsTeachModeStopBtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Teach Mode Stop</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_20">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="robotStatusIndicationLedLay_3"/>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Robot Teach Mode Status</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
